src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -1240,6 +1243,12 @@ def aks_update(cmd,     # pylint: disable=too-many-statements,too-many-branches,
     if attach_acr and detach_acr:
         raise CLIError('Cannot specify "--attach-acr" and "--detach-acr" at the same time.')
 
+    if uptime_sla:
```

Comment:
Not blocking but my preference is you create a new helper func (in a different file) that handles all SLA specific code. It is not blocking because this is only a few lines of code but I am guessing this code will grow and having it separated is preferable

Reply:
Actually, I see it is in both create and update - would really prefer if it it was in a short function that sets and returns the MC

Reply:
Just 3 lines of code, directly setting is not bad as well.